src/components/NewsFeed/MakePost.jsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import PropTypes from "prop-types";
 import firebaseWrapper from "../../firebaseWrapper";
 import "./MakePost.css";
+import PostImage from "../NotificationPanel/post.png";
```

Comment:
once you make the makePost function take in a type parameter, this import won't be needed